### PR TITLE
correct code that missing from rebase

### PR DESCRIPTION
**Pull request by canyugs**

### CODE REVIEW

File: extensions/chat/common/client.mjs [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ChatOverlay extends Overlay {
 
   // TODO(zeze-zeze): Expand on this class (jsdoc and implementation) and
   // implement onPre/PostShow/Hide() as needed.
-};
+}
```
Comment (john0312) [inline extensions/chat/common/client.mjs:17]: Is this correct?

File: extensions/chat/common/client.mjs [hunk 2]
```diff
@@ -95,26 +95,46 @@ Usage: !/<Command> <arg1> <arg2> ...
         document.getElementById('message_history').innerHTML += '<span>Invalid Command</span><br>';
         this.listCommand();
       }
-    };
+    }
```
Comment (john0312) [inline extensions/chat/common/client.mjs:98]: Not sure if this is the correct style. Will double check later.

File: extensions/chat/common/client.mjs [hunk 3]
```diff
@@ -95,26 +95,46 @@ Usage: !/<Command> <arg1> <arg2> ...
...
+   * @onExtensionBroadcast
+   * @param {object} arg - Information of the broadcast message
+  */
   onExtensionBroadcast(arg) {
     arg.msg_from = $('<div/>').text(arg.msg_from).html();
     arg.msg = $('<div/>').text(arg.msg).html();
     document.getElementById('message_history').innerHTML += '<span>' + arg.msg_from + ': ' + arg.msg + '</span><br>';
   }
-};
+}
```
Comment (john0312) [inline extensions/chat/common/client.mjs:138]: Same as above, I'm not sure.

File: extensions/chat/common/client.mjs (hunk 1 above)
Comment (canyugs) [reply extensions/chat/common/client.mjs:17]: sorry for none description 
all fixed by eslint --fix

File: extensions/chat/common/client.mjs (hunk 2 above)
Comment (canyugs) [reply extensions/chat/common/client.mjs:98]: sorry for no PR description
all fixed by eslint --fix

File: extensions/chat/common/client.mjs (hunk 3 above)
Comment (canyugs) [reply extensions/chat/common/client.mjs:138]: sorry for no PR description
all fixed by eslint --fix

**Pull request merged**